src/lamplib/src/genny/cli.py

```diff
@@ -250,51 +266,83 @@ def benchmark_test(ctx: click.Context) -> None:
 
 
 @cli.command(
-    name="workload",
-    help=(
-        "Actually calls the underlying genny binary called genny_core "
-        "which is installed by default to 'dist'. "
-        "Pass '--' to pass additional args to genny_core, e.g."
-        "run-genny workload -- run -w path_to_yaml -u mongodb://localhost:271017"
-    ),
+    name="workload", help=("Actually run a workload and place results in `build/WorkloadOutput`."),
```

Comment:
Nope it's removed and not necessary! I kinda feel like the options to the core binary are just internal implementation details of the python CLI.

Reply:
> the options to the core binary are just internal implementation details

Ya very good point. Now that we've got the `debug`/`hang` thing plus the curator handler there's very little reason for anyone to interact with `genny_core` directly. Let's hide it

(Maybe even consider setting a `GENNY_RUNNING_FROM_PYTHON` env var in the python (but with a non-terrible name). Then `genny_core` looks for that env var and barfs or yells if it doesn't see it. That may be too cheeky, but it would set the expectation correctly.)

Resolving the convo. This is good how it is.